src/i965_device_info.c

```diff
@@ -440,6 +441,7 @@ static struct hw_codec_info kbl_hw_codec_info = {
     .render_init = gen9_render_init,
     .post_processing_context_init = gen9_post_processing_context_init,
     .max_resolution = gen9_max_resolution,
+    .preinit_hw_codec = gen9_hw_codec_preinit,
```

Comment:
@seanvk Yakui filed https://github.com/01org/intel-vaapi-driver/issues/35 to track this issue.